src/T2DMap.cpp

```diff
@@ -4535,6 +4535,29 @@ void T2DMap::wheelEvent(QWheelEvent* e)
     e->ignore();
 }
 
+void T2DMap::changeMapperSettings(int roomSize, int lineSize, bool round, bool info, bool ids, bool viewOnly, bool showPanel)
+{
+  if (mpHost) {
+    this->mBubbleMode = round;
+    mpHost->mBubbleMode = this->mBubbleMode;
+    this->mShowInfo = info;
+    mpHost->mShowInfo = this->mShowInfo;
+    this->mShowRoomID = ids;
+    mpHost->mShowRoomID = this->mShowRoomID;
+    mpHost->mShowPanel = showPanel;
+    this->setMapViewOnly(viewOnly);
```

Comment:
You need to add this method to the list of - I guess (under the `private:`) methods listed in the header file.